From f113b4b0478cab45abc1e01bb04f79444a9a7d68 Mon Sep 17 00:00:00 2001
From: Peter Hutterer <peter.hutterer@who-t.net>
Date: Thu, 7 Jun 2012 12:00:49 +1000
Subject: [PATCH] Don't skip the prefix for udev's libdir when installing into
 /usr

As of F17, lib is /usr/lib, so don't mangle the libdir and just install into
the prefix

Signed-off-by: Peter Hutterer <peter.hutterer@who-t.net>
---
 configure.ac |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index d2fb0b6..cdf4cd8 100644
--- a/configure.ac
+++ b/configure.ac
@@ -94,7 +94,7 @@ AC_SUBST(HAL_FDI_DIR)
 
 case $host_os in
   linux*)
-    if test "x$prefix" = "xNONE" -o "x$prefix" = "x/usr" -o "x$prefix" = "x/usr/local" ; then
+    if test "x$prefix" = "xNONE" -o "x$prefix" = "x/usr/local" ; then
       slashlibdir=/lib
     else
       slashlibdir=$prefix/lib
-- 
1.7.10.2

